src/pages/filter/index.tsx

```
@@ -1,291 +1,317 @@
+                        const isAccessible = accessibleMetricDbColumns.has(metric.db_column);
+                        const fullRange = metricFullRanges?.[metric.db_column];
+                        const currentRange = filterSettings?.metricRanges?.[metric.db_column] || [null, null];
+                        
+                        // Determine absolute min/max for the slider, defaulting if range not loaded/valid
+                        const [absoluteMin, absoluteMax] = (fullRange && isValidNumber(fullRange[0]) && isValidNumber(fullRange[1])) 
+                                                            ? fullRange 
+                                                            : [0, 100]; // Fallback range for slider if API range is bad/missing
+                        
+                        const [currentMin, currentMax] = currentRange;
+                        
+                        const isRangeDataActuallyLoaded = metricFullRanges.hasOwnProperty(metric.db_column) && !loadingRanges;
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)